resources/keywords/deploy_instance_dev.resource

```diff
@@ -78,7 +78,8 @@ User Filters And Selects The Created Database Instance
 Instance Is Available In Selection Table
     ${instFound}=    Run Keyword And Return Status    Instance Can Be Selected
     Run Keyword If    not ${instFound}
-    ...    Reload Page
+    ...    Run Keywords
+    ...    Reload Page    AND    Fail
```

Comment:
Why do we need to refresh the page if the test fails?

Reply:
This is a edge case recursion scenario. After we provisioned the database instance, the instance table on developer view might not be refreshed instantly even though the process is success. To refresh the instance table, we are reloading the page and failing this keyword intentionally on line 82. So that, the recursion will occur to execute the same keyword.
If the instance table refreshed and instance provisioned available, then line 79 will return the status as "True". This will stop the recursion.
Overall we are performing recursion for 5 times with 2s delay, Even after if the provisioned instance not available then this test considered to failed.